_content/blog/2025-08-12-Oskari_in_a_container_hosting_on_a_cloud.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title:  "Oskari in container and hosting on a cloud platform"
+excerpt: "Interested in running Oskari as a Docker image or hosting in AWS? Look no further!"
+date:   2025-08-12  10:00:00 +0300
+categories: [news, container, cloud, developing]
+tags:
+  - blog
+
+---
+
+# Oskari in container and hosting on a cloud platform
+
+Some organizations have begun to use Oskari in a container on a cloud platform. For example, the city of Tampere uses Oskari as a container in Amazon's AWS service. The topic was discussed at the meeting of Oskari JDF in June and in this article we will cover the topic at a general level.
+
+# What are containers & cloud platforms?
+
+The most well-known cloud services at the moment are probably Amazon's AWS and Microsoft's Azure. They host services set up by their customers and offer various additional services related to the hosting. Often the customer buys a dedicated server for their service, but it is also possible to put the service in a container that is set up on a shared server. Doing so the price of the service can be reduced.
+
+Another option is to maintain the services in the organization’s own data center. If the organization has a data center ready and empty servers there, the services can of course be set up there. In an organization’s own data center services are (probably) physically close and setting up a new server involves manual labour with cables and computers, whereas with cloud platforms the same result can be achieved with a couple of clicks. This makes the cloud platforms a quick way to test the latest Oskari version, for example.
+
+When a service is set up on a cloud platform, the service provider is largely responsible for its functionality. If the organization faces problems with its own data center with Linux OS or the hardware, the organization is on its own.
+
+Running software in a container refers to exporting software to a Docker image. Docker is a software platform that uses containers to package and run applications. The containers include in one file everything an application needs to run: code, libraries, system tools, required libraries etc. This makes the development, testing and deployment of a software easy. Containers also provide a standardized way to package and distribute software.
+
+Cloud migrations may be driven by the need to separate public and government-use materials. On a cloud platform, public and intranet environments can be isolated from each other. 
```

Comment:
Things can be isolated without cloud as well, but maybe it's a clearer separation?

```suggestion
Cloud migrations may be driven by the need to separate public and government-use materials. Having a public facing cloud service and an on-premise internal service creates a very clear separation between the two use cases.
```